deepspeed/ops/transformer/inference/transformer_inference.py

```diff
@@ -589,7 +592,8 @@ def forward(ctx,
                 mlp_gemm_func,
                 fused_gemm_gelu,
                 vector_matmul_func,
-                bias_residual_func):
+                bias_residual_func,
+                activation_func_type=ActivationFuncType.GELU):
```

Comment:
I think this argument (`activation_func_type`) is not used anywhere, right? We can just remove it

Reply:
`activaiton_func_type` is passed to `mlp_gemm_func` so we can decide which activation kernel to launch in MLP.